rfc/0054-sdk3-management-apis.md

```diff
@@ -1494,6 +1530,192 @@ Map where top level keys are dataverse names, and values are a map of dataset na
 
     GET http://localhost:8095/analytics/node/agg/stats/remaining
 
+## CreateLink
+
+Creates a new link.
+
+### Signature
+
+```
+void CreateLink(AnalyticsLink link, [options])
+```
+
+### Parameters
+
+* Required:
+
+  * `link`: `AnalyticsLink` - the link to be created.
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+
+### Returns
+
+Nothing
+
+### Throws
+
+* `LinkAlreadyExistsException`
+
+* `AnalyticsScopeNotFoundException`
```

Comment:
There's an existing `DataverseNotFoundException` that seems to be mapped to the same server error code (24034)... Should we reuse it?

https://docs.couchbase.com/server/current/analytics/error-codes.html